integration_tests/samples/socket_mode/bolt_adapter/aiohttp.py

```diff
@@ -0,0 +1,107 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../../..")
+# ------------------
+
+import json
+import logging
+import os
+import asyncio
+from typing import Optional
+from time import time
+from slack_sdk.socket_mode.response import SocketModeResponse
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.socket_mode.aiohttp import SocketModeClient
+
+from slack_bolt import App
+from slack_bolt.request import BoltRequest
+from slack_bolt.response import BoltResponse
+
+
+class SocketModeAdapter:
+    app: App
+
+    def __init__(self, app: App):
+        self.app = app
+        # TODO: remove this once Bolt allow skipping this
+        if self.app._signing_secret is None:
+            self.app._signing_secret = "just-for-workaround"
+        from slack_sdk.signature import SignatureVerifier
+
+        self.signature_verifier = SignatureVerifier(
+            signing_secret=self.app._signing_secret
+        )
+
+    async def listener(self, client: SocketModeClient, req: SocketModeRequest):
+        start = time()
+        bolt_req: BoltRequest = self._build_bolt_request(req)
+        bolt_resp: BoltResponse = self.app.dispatch(bolt_req)
+        if bolt_resp.status == 200:
+            if bolt_resp.body is None or len(bolt_resp.body) == 0:
+                await client.send_socket_mode_response(
+                    SocketModeResponse(envelope_id=req.envelope_id)
+                )
+            elif bolt_resp.body.startswith("{"):
+                await client.send_socket_mode_response(
+                    SocketModeResponse(
+                        envelope_id=req.envelope_id, payload=bolt_resp.body,
+                    )
+                )
+            if client.logger.level <= logging.DEBUG:
+                spent_time = int((time() - start) * 1000)
+                client.logger.debug(f"Response time: {spent_time} milliseconds")
+        else:
+            client.logger.info(
+                f"Unsuccessful Bolt execution result (status: {bolt_resp.status}, body: {bolt_resp.body})"
+            )
+
+    def _build_bolt_request(self, req: SocketModeRequest) -> BoltRequest:
+        # TODO: remove this signature generation once Bolt allow skipping this
+        raw_body: str = json.dumps(req.payload)
+        timestamp: str = str(int(time()))
+        signature: str = self.signature_verifier.generate_signature(
+            timestamp=timestamp, body=raw_body
+        )
+        return BoltRequest(
+            body=raw_body,
+            headers={
+                "Content-Type": "application/json",
+                "X-Slack-Request-Timestamp": timestamp,
+                "X-Slack-Signature": signature,
+            },
+        )
+
+
+class SocketModeApp:
+    app: App
+    app_token: str
+    client: SocketModeClient
+
+    def __init__(
+        self, app: App, app_token: Optional[str] = None,
```

Comment:
I suggest using `app_token` over `token` for key name also in other Bolt framework (as we are easily confused with the difference between OAuth tokens and app tokens)